pages/orgs/[org]/domains/[domain]/index.js

```diff
@@ -108,39 +110,86 @@ function OrgDomainsList() {
   };
 
   const fetchDomainData = (page = 1) => {
-    fetch("/api/domains/" + domain + "?includeConcepts=true&page=" + page)
-      .then((response) => {
-        if (!response.ok) {
-          throw new Error(`HTTP error! Status: ${response.status}`);
-        }
-        return response.json();
-      })
-      .then((data) => {
-        if (data) {
-          setDomainData(data);
-          setSubDomainData(data.data?.subdomains);
-          let filteredConcepts = data?.data?.concepts?.filter(
-            (concept) => concept.type === "Concept"
-          );
-          if (data?.data?.subdomains && data?.data?.subdomains.length > 0) {
-            setSelectedSubdomain(data?.data?.subdomains[0]?.url);
-            fetchConcepts(data?.data?.subdomains[0]?.url);
+    // Check if the domain ID is `hpt-non-pharmaceuticals` or `hpt-pharmaceuticals`
+    let cascadeUrl = null;
+
+    if (domain === "hpt-non-pharmaceuticals") {
+      // Use cascade query URL for HPT-non-pharmaceuticals
+      cascadeUrl = `${API_BASE_URL}/orgs/MOH-KENYA/sources/PPB/concepts/10-00000-02/$cascade/?cascadeLevels=1&method=sourceToConcepts&view=hierarchy&reverse=true&includeRetired=false`;
+    } else if (domain === "hpt-pharmaceuticals") {
+      // Use cascade query URL for HPT-pharmaceuticals
+      cascadeUrl = `${API_BASE_URL}/orgs/MOH-KENYA/sources/PPB/concepts/10-00000-01/$cascade/?cascadeLevels=1&method=sourceToConcepts&view=hierarchy&reverse=true&includeRetired=false`;
```

Comment:
@lucyjemutai  - Can we extract this to the domains json and reference them here as an attribute of a domain instead of directly referencing them this way? like `domain.cascadeUrl`